sdk/src/realtime/composable.ts

```diff
@@ -64,7 +64,7 @@ export function realtime(config: WebSocketConfig = {}) {
 				if (token) url.searchParams.set('access_token', token);
 			}
 
-			return url;
+			return url.toString();
```

Comment:
The `withStrictAuth` helper should still return an URL.
```suggestion
			return url;
```

Reply:
Felt a bit cleaner to do the transformation inside the helper. The helper is not used elsewhere, so wouldn't break anything.
But happy to update if you prefer the `.toString()` call inside `WebSocket()` creation 👍